.github/CHANGELOG.md

```diff
@@ -3,13 +3,45 @@
 <h3>New features since last release</h3>
 
 * Adds the new template `AllSinglesDoubles` to prepare quantum states of molecules
-  using the `SingleExcitation` and `DoubleExcitation`. 
+  using the `SingleExcitation` and `DoubleExcitation` operations. 
   The new template reduces significantly the number of operations
   and the depth of the quantum circuit with respect to the traditional UCCSD
-  unitary. The performance of quantum chemistry simulations should benefit
-  from this new implementation. 
+  unitary.
   [(#1383)](https://github.com/PennyLaneAI/pennylane/pull/1383)
 
+  For example, consider the case of two particles and four qubits.
+  First, we define the Hartree-Fock initial state and generate all
+  possible single and double excitations. 
+
+  ```python
+  import pennylane as qml
+  from pennylane import numpy as np
+
+  electrons = 2
+  qubits = 4
+
+  hf_state = qml.qchem.hf_state(electrons, qubits)
+  singles, doubles = qml.qchem.excitations(electrons, qubits)
+  ```
+  Now we can use the template ``AllSinglesDoubles`` to define the
+  quantum circuit,
+
+  ```python
+  wires = range(qubits)
+
+  @qml.qnode(dev)
+  def circuit(weights, hf_state, singles, doubles):
+      qml.templates.AllSinglesDoubles(weights, wires, hf_state, singles, doubles)
+      return qml.expval(qml.PauliZ(0))
+
+  params = np.random.normal(0, np.pi, len(singles) + len(doubles))
+  circuit(params, hf_state, singles=singles, doubles=doubles)
```

Comment:
We can just write
`circuit(params, hf_state, singles, doubless)`
correct?

Reply:
Good point. I discussed this with Tom. Within the ``autograd`` if ``singles`` and``doubles`` are not passed as kwargs, their type is changed automatically inside the qnode to Numpy arrays which causes an error in the template.  See [here](https://github.com/PennyLaneAI/pennylane/blob/aba87f1387234cdd7e4ece841e8c4311ceda05c0/pennylane/qnode.py#L483)

Reply:
Flyby comment: I think a more long term solution here is to specify `requires_grad=False` on both single and doubles, then it doesn't matter where how they are used in the signature

Reply:
I hacked a check in the template to try this but it did not work.

Reply:
I think what Josh means is that if `singles` and `doubles` is a `numpy` array, a safe (but slightly ugly) option would be to cast it into `pennylane.numpy` and set `requires_grad=False`. This will be safe even if a user forgets the kwarg passing... 

But it is a bit confusing why one would do that, so I am not against the way you do it, and if users change the signature and get errors we have another incentive to change PennyLane's internal casting behaviour!

Reply:
Yes, I agree.